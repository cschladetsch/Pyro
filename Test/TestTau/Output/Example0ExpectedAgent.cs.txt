using Pyro.Network;

namespace MyNamespace.Agent {
    interface IFooAgent
        : IAgentBase<IFooAgent> {
        string Name { get return GetName(); set return SetName(value); }
        
        string GetName();
        bool SetName(string name);
        int Sum(int a, int b);
    }
}
