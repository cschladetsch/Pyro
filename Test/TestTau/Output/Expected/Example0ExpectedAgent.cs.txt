using Pyro.Network;

namespace MyNamespace.Agent {
    interface IFooAgent : IAgentBase {
        string Name { get; set; }
        
        int Sum(int a, int b);
    }
}
